### DISCUSSION TITLE
Can't see the achievement section in my profile even though I have enabled it from the Settings

### DISCUSSION THREAD

**Discussion by isyneha** [community/community#37455] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/79058289/198213038-cbde0828-ff7b-4bde-9dca-f0b4b2d0ec99.png)

**Comment by radar-blip** [comment]
Did you remember to `Update preferences` ?

![show_achievements](https://user-images.githubusercontent.com/39196767/198233068-94e248ad-c5ac-4e3a-8530-09d342924657.png)

**Reply by isyneha** [reply]
Yes, I did that but still, it is not showing up. Though it used to show up earlier.

**Reply by nikolast178** [reply]
I have the same problem. I try to use "/**user-name**?tab=achievements" but I get **404**.

**Comment by MizunaRei** [comment]
I hide my only one achievement by clicking the eye icon on achievement detail page. The achievement section has gone from my profile page.

**Reply by isyneha** [reply]
Is there a way to bring it back?

**Reply by radar-blip** [reply]
Yes, you can bring it back. You just re-click the eye icon.
But since you dont have any achievements showing, I dont know how you would get there to begin with.

Below is the new setting for showing individual achievements (might have something to do with yours not showing).

![hideach](https://user-images.githubusercontent.com/39196767/198508178-d4206b5c-5894-4fc3-8eb3-31941bbffee7.png)

**Reply by isyneha** [reply]
Hey, thanks for your help. I have found a way to bring it back and posted here as well.

**Reply by radar-blip** [reply]
Yes I know lol 👍🏻

**Comment by isyneha** [comment, marked as answer]
Well, I found a way to solve this!
 
Simply type this in the browser-

https://github.com/github-username?tab=achievements

A page will show up with your achievements. Simply, unhide your achievement from there.

**Reply by radar-blip** [reply]
We must have been typing at same time. Anyway, glad you found a way!

By the way, that link you posted isn't working (for me anyway).

**Reply by isyneha** [reply]
Yes, I think so. Just replace "github-username" with your own Github username. That will work I guess.

**Reply by radar-blip** [reply]
Yes, I figured that out, thanks!

**Reply by kumar-chetan** [reply]
Copy paste this 
 https://github.com/kuldeep1A?tab=achievements
 you get your Achievements back on your Profile 
 Thanks me later 😁

**Reply by Brigoliboon** [reply]
I landed on a 404 error with my link: https://github.com/Brigoliboon?tab=achievements
I don't really know what's the problem because I enabled it from the settings and I just accessed my achievements panel recently

**Reply by TheZincTZ** [reply]
i still cant see mine

**Reply by Sakith2820** [reply]
I have  try  [https://github.com/sakith2820?tab=achievements](https://github.com/sakith2820?tab=achievements ) but it landing on  404 error
![Uploading Screenshot 2025-09-13 074700.png…]()

**Reply by EHEGUY** [reply]
same landing a 404 page any solution to this ?

**Comment by jfullstackdev** [comment]
and the good thing is that you can view others too, it's a public endpoint, I tried mine and still working

**Comment by yzc2020** [comment]
My badges aren't hidden, but my entire section is not showing up:

![image](https://user-images.githubusercontent.com/53286102/226373698-4390c7ae-9537-4dd1-a256-5d989ce91823.png)

It's like a bug of GitHub: https://github.com/yzc2020?tab=achievements

**Reply by jisaso** [reply]
Same issue

**Comment by asudox** [comment]


**Reply by Kuldeep1A** [reply]
because you does not achieve any achievement. If you achieve any achievement it showing your achievement in achievement section

**Reply by Brigoliboon** [reply]
we have the same issue.  Is the problem resolved?

**Reply by sai-123-code** [reply]
yes ,if you don't have achievements section then it might be either you dont have achievements or you hide that section and for me pull shark achievements got unlocked after removing the branch after pr was approved 
<img width="523" alt="image" src="https://github.com/user-attachments/assets/79cefef9-c780-4f0a-a560-8c41c870690e" />
